django/db/migrations/loader.py

```diff
@@ -316,6 +319,34 @@ def _reraise_missing_dependency(migration, missing, exc):
                         # "child" is not in there.
                         _reraise_missing_dependency(migration, child, e)
 
+    def check_consistent_history(self, connection):
+        """
+        Checks that all migrations are applied after their dependencies. Raises
+        InconsistentMigrationHistory exception when migration in applied before
+        it's dependency
+        """
+        recorder = MigrationRecorder(connection)
+        applied = recorder.applied_migrations()
+        for migration in applied:
+            # If we don't know anything about this migration we skip it
+            if migration not in self.graph.nodes:
+                continue
+            # Check only parents from the same app
+            unapplied_parents = [
+                x.key for x in self.graph.node_map[migration].parents
+                if x not in applied and x[0] == migration[0]
+            ]
```

Comment:
It probably would not be in a graph if it was disabled. Also, I'm overriding `MIGRATION_MODULES`, so in the loader I don't know which apps are disabled. Even more, it considers all application that are set to `None` in `runtests.py` to be migrated, because of this there is no error, when I add `admin` to it.